From cb0dc03d7d1f584f3c17ab23631077a4dc28c697 Mon Sep 17 00:00:00 2001
From: Carlos Soriano <csoriano1618@gmail.com>
Date: Thu, 30 Nov 2017 09:33:51 +0000
Subject: [PATCH] Merge branch '134-xf86back-and-xf86forward-not-working-anymore' into 'master'

Resolve "XF86Back and XF86Forward not working anymore"

Closes #134

See merge request GNOME/nautilus!32

(cherry picked from commit 3ba92d47e44049eaf81fb3d885a2145eb3686a3a)

2c5882aa build: Detect X11/XF86keysym.h to enable extra keys
bdd1c3b0 build: Fix indent style
---
 config.h.meson | 1 +
 meson.build    | 6 +++++-
 2 files changed, 6 insertions(+), 1 deletion(-)

diff --git a/config.h.meson b/config.h.meson
index 4f5cb58..fa8c131 100644
--- a/config.h.meson
+++ b/config.h.meson
@@ -4,6 +4,7 @@
 #mesondefine HAVE_EXEMPI
 #mesondefine HAVE_EXIF
 #mesondefine HAVE_SELINUX
+#mesondefine HAVE_X11_XF86KEYSYM_H
 #mesondefine ENABLE_DESKTOP
 #mesondefine ENABLE_PACKAGEKIT
 #mesondefine LOCALEDIR
diff --git a/meson.build b/meson.build
index 7b8f15c..930a939 100644
--- a/meson.build
+++ b/meson.build
@@ -83,7 +83,11 @@ endif
 
 tracker_sparql = dependency ('tracker-sparql-2.0', required: false)
 if not tracker_sparql.found()
-  tracker_sparql = dependency ('tracker-sparql-1.0')
+    tracker_sparql = dependency ('tracker-sparql-1.0')
+endif
+
+if cc.has_header ('X11/XF86keysym.h')
+    conf.set10 ('HAVE_X11_XF86KEYSYM_H', true)
 endif
 
 if get_option ('enable-xmp')
--
libgit2 0.26.0

